source/_patterns/01-elements/link/link.scss

```diff
@@ -31,46 +36,43 @@
 		}
 	}
 
-	// Sizes
-	&%size-Small {
-		&:not(.is-icon-text-replace) {
-			font-size: to-rem($pxValue: 14);
-		}
-
-		&[rel],
-		&[data-icon],
-		&[data-icon-before] {
-			@include icon-meta(20);
-		}
-
-		&[data-icon-after]::after {
-			@include icon-meta($size: 20, $position: "after");
-		}
+	&[data-content] {
+		--icon-font-family: var(--db-base-icon-font-family);
+		--icon-font-size: var(--db-base-icon-font-size);
 	}
 
-	&[data-size="small"] {
-		@extend %size-Small;
+	&[data-content="internal"] {
+		@include icon(glyph(link), 24, "outline", "after");
 	}
 
-	// Define the available icons
-	&[rel="configuration"] {
-		@include icon(glyph(settings), 24, "outline", $partial: $partial);
+	&[data-content="external"] {
+		@include icon(glyph(link-external), 24, "outline", "after");
 	}
 
-	&[rel="messages"] {
-		@include icon(glyph(chat), 24, "outline", $partial: $partial);
+	&[data-content="internal"],
+	&[data-content="external"] {
+		&::after {
+			--icon-margin-before: var(--db-spacing-fixed-2xs);
+			margin-top: -3px;
+		}
+
+		&:focus-visible {
+			outline-offset: -1px;
+			padding: 0 2px;
+		}
 	}
 
-	&[rel="account"] {
-		@include icon(glyph(account), 24, "outline", $partial: $partial);
+	&[data-variant="primary"] {
+		@extend %db-primary-text-ia;
```

Comment:
The build step outputs the following error:
```
Use "@extend %db-primary-text-ia !optional" to avoid this error.
   ╷
66 │         @extend %db-primary-text-ia;
   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  source/_patterns/01-elements/link/link.scss 66:3  root stylesheet
```

Reply:
this got fixed by https://github.com/db-ui/core/pull/89/commits/99f933f323b648fc8c9b21ff3c862353162ebfe0